exercises/binary-search-tree/binary_search_tree.py

```diff
@@ -0,0 +1,8 @@
+class TreeNode(object):
+    def __init__(self, value):
+        pass
```

Comment:
Can you add `self.value = value` here? I feel like if we assume that something exists in the tests, then we should make it clear in the solution template.

Reply:
Done